src/frontend/src/env/rest/etherscan.env.ts

```diff
@@ -1 +1,5 @@
+import { BETA, PROD } from '$lib/constants/app.constants';
+
 export const ETHERSCAN_API_KEY = import.meta.env.VITE_ETHERSCAN_API_KEY;
+
+export const ETHERSCAN_MAX_CALLS_PER_SECOND = BETA || PROD ? 10 : 5;
```

Comment:
Staging should not be considered as prod?

Reply:
Well, for staging we have an API key that is the Free plan, hence the lower limit. Unless we decide to change it, but I think this should be another topic, offline eheh